### ISSUE TITLE
[Enhance] Spark Accordion Item will use <button> as their triggering elements - Angular

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#2939]
**User Story:**
As Spark we want to update Spark Accordions from a link to a button so that it is accessible.

**Notes:**
Currently spark's accordions use `<a>` when, since they don't navigate, should be `<button>`. See rm-12 in the a11y audit.

**AC:**
Angular Accordion
- will see `<a>` be changed to use `<button>`
- will see aria-expanded

aria-expanded: this attribute is on the trigger element (the button). Its value is set to "true" when the element is expanded and is set to "false" when the element is collapsed.
